src/vit-servicing-station/vit-servicing-station-lib/src/db/migrations.rs

```diff
@@ -1,20 +1,46 @@
+use tempfile::TempDir;
+
 use crate::db::DbConnection;
 
+const PG_INITIAL_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/00000000000000_diesel_initial_setup/up.sql");
+const PG_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/2022-11-03-210841_setup_db/up.sql");
+const SQLITE_SETUP_UP: &str =
+    include_str!("../../migrations/sqlite/2020-05-22-112032_setup_db/up.sql");
+
 pub fn initialize_db_with_migration(db_conn: &DbConnection) {
     // Discard all output
     let mut sink = std::io::sink();
 
     match db_conn {
-        DbConnection::Sqlite(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/sqlite"),
-            &mut sink,
-        ),
-        DbConnection::Postgres(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/postgres"),
-            &mut sink,
-        ),
+        DbConnection::Sqlite(db_conn) => {
+            let t = TempDir::new().unwrap();
+
+            write_tmp_migration(&t, "2020-05-22-112032_setup_db", SQLITE_SETUP_UP);
```

Comment:
what does this represent? _"2020-05-22-112032_setup_db"_

does it refer to something on your local machine?

Reply:
That's the a directory containing SQL db migration. It's created by Diesel when you generate a new migration.

Reply:
PS: https://github.com/input-output-hk/catalyst-core/tree/main/src/vit-servicing-station/vit-servicing-station-lib/migrations

Reply:
👍